### PR TITLE
added port option for postgres connection

### PR DESCRIPTION
**Pull request by sHtev**
Just uses the port in the DATABASES setting - defaults ok when not set explicitly in settings.

### CODE REVIEW

File: zap/backends/postgresql.py [hunk 1]
```diff
@@ -28,10 +28,13 @@ def _psql(self, *command):
         ''' Run a command via psql as the postgres user '''
         if self.debug:
             sys.stderr.write('psql -c "' + ' '.join(command) + '"\n')
-        base_command = ['sudo', '-u', 'postgres', 'psql', '-c']
+        port_string = '{port}'.format(port=self.port)
+        base_command = ['sudo', '-u', 'postgres',
+                        'psql', '-p', port_string, '-c']
```
Comment (alex2) [inline zap/backends/postgresql.py]: Does this definitely still work when settings.DATABASES['PORT'] is an empty string that gets passed to psql?

Comment (sHtev) [inline zap/backends/postgresql.py]: I was testing with DJ and a DATABASE_URL env variable, so I'm guessing not.

Will add a test for an empty port string, if that suffices, not sure if you want a more elegant way to construct the command string at this point.

Comment (alex2) [inline zap/backends/postgresql.py]: Hmm, probs best to omit -p altogether if self.port is falsey. I'll leave it up to you to concoct some fancy one-liner to do the job :smile:

**Comment by sHtev** [comment]
Have updated to test for empty port and build command string list more sensibly.
